liveDeploys/elixir.html

```diff
@@ -29,28 +26,30 @@ <h5>Elixir resources using Bioschema's markup</h5>
             <thead>
                 <tr>
                     <th>Name</th>
-                    <th>Profile</th>
-                    <th>Profile Version</th>
+                    <th class="profile-column">Profile</th>
+                    <th class="structured-data-column">Profile Version</th>
+                    <th class="structured-data-column">Page</th>
```

Comment:
Is page a web page? Not sure the column title is the right one

Reply:
Yes it is the web page with the deployed markup. Happy to rename to web page if you think that is clearer

Reply:
It is probably clear from the content so let's keep it short as it is now